### ISSUE TITLE
Accessibility Tab Questions

### ISSUE THREAD

**Issue by mattcowan** [mattcowan/ligatures-and-typographic-styles-for-wordpress#36]
Is the aria-label area still something we should have?
Does clear button confirmation belong on this tab? Probably not.
